src/utils/clmfates_interfaceMod.F90

```diff
@@ -1111,13 +1111,26 @@ subroutine wrap_update_hlmfates_dyn(this, nc, bounds_clump,      &
             this%fates(nc)%bc_out )
 
        !------------------------------------------------------------------------
-       ! Get FATES calculation of ligninNratio
+       ! FATES calculation of ligninNratio
        !------------------------------------------------------------------------
-       do s = 1, this%fates(nc)%nsites
-          c = this%f2hmap(nc)%fcolumn(s)
-          soilbiogeochem_carbonflux_inst%litr_lig_c_to_n_col(c) = &
+       ! If it's the first timestep of a restart and copy_fates_var = .false.
+       ! (this will happen in the first timestep of any restart)
+       ! then skip this variable because a more accurate value was obtained
+       ! from the restart file.
+       ! I had hoped that is_first_restart_step() alone would suffice here, but
+       ! is_first_restart_step() remained true for the whole first day in my
+       ! test. Hence I added the check for copy_fates_var which changes to
+       ! .true. the first time that we come through this code in a restart.
+       !------------------------------------------------------------------------
+       if (is_first_restart_step() .and. .not. copy_fates_var) then
+          copy_fates_var = .true.
```

Comment:
@rgknox @ekluzek 
This gets the MIMICS-FATES restart test to pass by not copying the variable from FATES when `is_first_restart_step = .true.` and using instead the corresponding CTSM variable that I sent to restart. For more, pls read the comment above the if statement.

My question is whether you consider this solution acceptable or would you rather determine why the FATES variable isn't restarting correctly?

Meanwhile I have submitted the test suites and will report results in the morning.

Reply:
So this PR is ready to merge unless anyone has concerns about my solution to the failing restart MIMICS-FATES test (my last commit).

Reply:
Sounds like we should maybe meet to go over this code. Sometime tomorrow work?

Reply:
This should be threadsafe as different threads would have different instances of the entire wrap_update_hlmfates_dyn subroutine. So that should be OK as long as it's not saved. We should maybe make sure we have a threading restart test though.

There is a save statement on the initialized variable and I wonder if that is threadsafe.

Reply:
Nope, @rgknox is correct, this is global so needs to be a module array based on the number of threads. Thanks for pointing that out!

Reply:
We met and decided that I will:
- confirm that other fates variables getting copied to ctsm variables have same values at restart as in a continuous run
- if so, look for the variable responsible for the failing restart in the calculation of our new variable `litt_flux_ligc_per_n`

Reply:
...I take that back. Follow-up with @rgknox suggests that I pursue the workaround for now.